Docs/source/running_cpp/parameters.rst

```diff
@@ -940,6 +940,11 @@ Numerics and algorithms
 * ``psatd.nox``, ``psatd.noy``, ``pstad.noz`` (`integer`) optional (default `16` for all)
     The order of accuracy of the spatial derivatives, when using the code compiled with a PSATD solver.
 
+* ``nx_guard_psatd``, ``ny_guard_psatd``, ``nz_guard_psatd`` (`integer`) optional
```

Comment:
For consistency with the other parameters, could this be:
``psatd.nx_guard``, ``psatd.ny_guard``, ``psatd.nz_guard``?

Reply:
Done in commit 4fcbac6900123363f58dc8.